pingdom/src/main/java/com/pulse/repository/AutheneticationRepository.java

```diff
@@ -0,0 +1,19 @@
+package com.pulse.repository;
+
+import com.pulse.payload.User;
+
+public class AutheneticationRepository {
```

Comment:
Don't need this. Pass the information from the UI API module and process it. DOn't process cookies